test/verify/check-system-info

```diff
@@ -481,7 +481,7 @@ class TestSystemInfo(MachineCase):
         self.restore_dir("/usr/local/bin")
 
         self.login_and_go('/system/hwinfo')
-        b.wait_visible('#hwinfo th:contains(CPU)')
+        b._wait_present('#hwinfo th:contains(CPU)')
```

Comment:
I think for this one, `wait_in_text("#hwinfo", "CPU")` would be sufficient. It's just to make sure that the table got rendered.